docs/k8s/index.mdx

```diff
@@ -29,77 +11,27 @@ applications run in Kubernetes environments. It supports both [Ingress CRDs](htt
 
 The ngrok Kubernetes Operator is deployed to a cluster and allows developers to self-service public endpoints to their apps and services using a shared ngrok account.
 
-The operator watches for Kubernetes objects and creates the corresponding ngrok endpoints and configurations.
-
-More details on how these are derived can be found [here](/docs/k8s/with-edges).
-
-Other ngrok features such as TCP Edges can be configured via [CRDs](/docs/k8s/crds).
-
-If you are looking to install the controller for the first time, see our [deployment-guide](/docs/k8s/deployment-guide/).
-If it's already installed and you are looking to configure ingress for an app or service, see our [user-guide](/docs/k8s/user-guide/).
-
-## Install
-
-Installing the ngrok Operator is easy using Helm.
-
-<Tabs groupId="k8s" queryString="k8s-install">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<K8sIngressInstallExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<K8sGatewayAPIInstallExample />
-	</TabItem>
-</Tabs>
-
-<br />
-
-## Example Usage
-
-This would create an Edge on with the Domain `example.ngrok.app` that routes traffic to the Kubernetes service `example-service` running on port 80.
-
-<Tabs groupId="k8s" queryString="k8s-basic">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<StaticDomainKubernetesExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<StaticDomainKubernetesGatewayExample />
-	</TabItem>
-</Tabs>
+## When should I use the Kubernetes Operator?
 
-## Route Traffic based on a path
+The Kubernetes Operator is a great tool to use when you want to use ngrok to connect to services running on a Kubernetes Cluster. If you are running Kubernetes and do not want to use the ngrok SDKs, we recommend using the Operator
 
-This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from example.ngrok.app/foo to the Kubernetes service `example-service` running on port 80.
+## Should I use the Ingress Controller or the Gateway API?
 
-<Tabs groupId="k8s" queryString="k8s-basic-path">
-	<TabItem value="ingress" label="Ingress Controller" default>
-		<StaticDomainKubernetesPathExample />
-	</TabItem>
-	<TabItem value="gatewayAPI" label="Gateway API">
-		<StaticDomainKubernetesGatewayPathExample />
-	</TabItem>
-</Tabs>
+The Operator supports both Kubernetes Ingress and the Gateway API. These two different solutions have differences and similarities.
 
-## Modify Headers
+### What is the same?
```

Comment:
not sure we need the section headers here. feels like we can just list the differences and similarities.

Reply:
This section has been removed.